presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -1237,8 +1256,8 @@ public static String metadataLocation(Table icebergTable)
 
     /**
      * Get the data location for target {@link Table},
-     *  considering iceberg table properties {@code WRITE_DATA_LOCATION}, {@code OBJECT_STORE_PATH} and {@code WRITE_FOLDER_STORAGE_LOCATION}
-     * */
+     * considering iceberg table properties {@code WRITE_DATA_LOCATION}, {@code OBJECT_STORE_PATH} and {@code WRITE_FOLDER_STORAGE_LOCATION}
```

Comment:
I know the comments were improperly formatted before, but can you revert this?  It would just muddle the git blame.

Reply:
IntelliJ IDE likes to auto-format these...maybe I'll open a separate PR to apply formatting to the entire connector. This seems to happen quite often